app.js

```diff
@@ -0,0 +1,66 @@
+function getPlayerMove() {
+  //Gets player input as string
+  var playerinput = prompt("Enter your move: rock, paper, scissors")
+  //Converts player input into a number between 0 and 2 with the scheme:
+  //rock = 0
+  //paper = 1
+  //scissors = 2
+  switch (playerinput) {
+    case 'rock':
+      return '0';
+      break;
+    case 'paper':
+      return '1';
+      break;
+    case 'scissors':
+      return '2';
+    default:
+      throw 'invalidInputError';
```

Comment:
This error is only thrown in the console - the user doesn't actually get to see it. It would be better to have an alert saying "Invalid input: Please enter rock, paper, or scissors" or something.